src/sonic-yang-models/yang-models/sonic-vrf.yang

```diff
@@ -0,0 +1,49 @@
+module sonic-vrf {
+	namespace "http://github.com/Azure/sonic-vrf";
+	prefix vrf;
+	
+	import sonic-extension {
+		prefix sonic-ext;
+	}
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC VRF";
+
+	revision 2021-03-30 {
+		description
+			"Initial revision.";
+	}
+
+	container sonic-vrf {
+		container VRF {
+			description "Vrf configuration.";			 
```

Comment:
Is mgmt vrf yang expected to be defined separately?

Reply:
Yes Prince. mgmt-vrf yang will be defined separately.

Reply:
Mgmt VRF PR : https://github.com/Azure/sonic-buildimage/pull/7252